content/apps/overview.md

```diff
@@ -75,7 +75,7 @@ Your app is good to go?
 
 - Discover our [extension approval requirements](/apps/publish-your-app.html#app-approval-requirements)
 - [Contact us](https://www.akeneo.com/contact/) to demonstrate your app
-- Read our [how to publish your app](https://apps.akeneo.com/how-submit-extension-akeneo-app-store) guide
+- Read our [how to publish your app](https://api.akeneo.com/apps/publish-your-app.html) guide
```

Comment:
It seems there is a issue with that page, when you go to https://api.akeneo.com/apps/homepage.html you do not have the submenu available either in the top menu or in the left. 
You have to follow the exact url to get there.
More over there is also a page https://api.akeneo.com/app-portal/publish-your-app.html which talk about App Approval Requirements and it could be a bit confusing